tests/test_utils.py

```diff
@@ -46,14 +46,49 @@ def test_imresize():
     )
     assert resized_img.shape == (1000, 500, 3)
 
-    # test for dtype conversion
+    # test for dtype conversion, pairs of
+    # (original type, converted type)
+    test_dtypes = [
```

Comment:
I do not understand how you define these pairs?

Reply:
Opencv resize function only works on a fixed set of np.dtype . Hence, for dtypes that cv2 do not work on, we need to convert it to the closest dtype. Here, the converted dtypes are selected such that it includes the precision range of original dtype. As for how we know which dtype is supported by opencv, I manually run for each dtype and check which one will crash.